geode-core/src/main/java/org/apache/geode/distributed/internal/locks/DLockService.java

```diff
@@ -1433,29 +1412,7 @@ public boolean lockInterruptibly(final Object name, final long waitTimeMillis,
         int lockId = -1;
         incActiveLocks();
 
-        int loopCount = 0;
         while (keepTrying) {
```

Comment:
This `keepTrying` thing is very unclear because it hides what the actual decision is based on. It seems like we keep trying until either the `waitLimit` has been exceeded or the lock has been acquired. Could we split `keepTrying` out into `boolean lockAcquired` and `boolean waitLimitExceeded`? Maybe the check for `waitTimeLimit` doesn't even need a variable but could be checked by the `while`?